src/agents/toolkits/site-spec.js

```diff
@@ -101,20 +89,21 @@ class SimpleSiteToolkit extends Toolkit {
 	];
 
 	getValues = () => {
+		const state = this.stateManager.getState();
 		return {
 			site: {
-				title: this.state.siteTitle,
```

Comment:
Again, rather than having a whole this.something.getSomething pattern here, it would be simpler to have a dynamic getter on the class, like: 

```
get state() {
  return this.stateManager.state;
}
```

Then, inside the stateManager, you could also do this:

```
get state() {
  // whatever logic you want
}
```


Then none of this code would have to change and you would reduce coupling on internal implementation details across the board.